compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/checkers/declaration/FirNotImplementedOverrideChecker.kt

```diff
@@ -190,21 +137,27 @@ object FirNotImplementedOverrideChecker : FirClassChecker() {
         }
     }
 
-    private fun FirCallableDeclaration<*>.isFromInterfaceOrEnum(context: CheckerContext): Boolean =
-        (getContainingClass(context) as? FirRegularClass)?.let { it.isInterface || it.isEnumClass } == true
+    private fun processMemberSymbols(
+        firClass: FirClass<*>,
+        sessionHolder: SessionHolder,
+        collect: (symbol: FirCallableSymbol<*>, implementationStatus: ImplementationStatus, isVisible: Boolean) -> Unit
```

Comment:
Nit: instead of `collect`, maybe `processor` aligns with `processMemberSymbols`.

Reply:
Sure thing!

Reply:
OK, I have refactored the code so that now there is no longer the `processMemberSymbol` function.